modules/monitoring/helm-values/grafana-values.yaml.tpl

```diff
@@ -9,6 +9,7 @@ persistence:
   type: pvc
   enabled: true
   size: 1Gi
+  storageClassName: gp2
```

Comment:
Would it make sense to use gp3 instead? (it sounds like a cheaper and newer option, but might not be supported by all components)